web/flat/index.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<html lang="fr">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AppBuild</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i,700,700i" rel="stylesheet">
+    <link rel="stylesheet" href="./css/app.css">
+  </head>
+  <body>
+    <section class="section-login">
+      <div class="row align-center align-middle">
+        <div class="columns small-10 medium-5 large-4 form-login text-center">
+          <header class="form-login__header">
+            <h1 class="logo"><img src="./img/logo.svg" alt=""></h1>
+            <h2 class="logo-baseline">by linkvalue</h2>
+          </header>
+          <form action="">
+            <div class="input-wrap">
+              <input type="email" id="email" class="input-wrap__input">
+              <label for="email" class="input-wrap__label">e-mail</label>
+              <i class="bar"></i>
+            </div>
+            <div class="input-wrap">
+              <input type="text" id="password" class="input-wrap__input">
+              <label for="password" class="input-wrap__label">mot de passe</label>
+              <i class="bar"></i>
+            </div>
+            <div class="form-login__actions text-center">
+              <a href="apps.html" class="button button_action button_confirm button_confirm-login" data-button-login><span>se connecter</span></a>
+              <a class="form-login__forgot-password" href="">mot de passe perdu ?</a>
```

Comment:
I'm sure you already know it, but this application supports both English and French languages